libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -755,10 +755,21 @@ - (BOOL)loadUserAccountFromFile:(NSString *)filePath encrypted:(BOOL)encrypted a
         
         @try {
             SFUserAccount *decryptedAccount = [NSKeyedUnarchiver unarchiveObjectWithData:decryptedArchiveData];
-            if (account) {
-                *account = decryptedAccount;
+            
+            // On iOS9, it won't throw an exception, but will return nil instead.
+            if (decryptedAccount) {
+                if (account) {
+                    *account = decryptedAccount;
+                }
+                return YES;
+            } else {
+                if (error) {
+                    *error = [NSError errorWithDomain:SFUserAccountManagerErrorDomain
+                                                 code:SFUserAccountManagerCannotReadDecryptedArchive
+                                             userInfo:@{ NSLocalizedDescriptionKey : reason} ];
+                }
+                return NO;
             }
```

Comment:
This code path is only exercised from the test apparently and also need the special iOS9 handling.